schemas/2015-09-01/Microsoft.MySql.json

```diff
@@ -0,0 +1,489 @@
+﻿{
+  "id": "http://schema.management.azure.com/schemas/2015-09-01/Microsoft.Sql.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.MySQL",
+  "description": "Microsoft MySQL Database Resource Types",
+  "definitions": {
+    "serversParameters": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string"
+        },
+        "allowAzureServices": {
+          "type": "boolean"
+        },
+        "dailyBackupTimeInHour": {
+          "type": "integer"
+        },
+        "options": {
+          "type": "object",
+          "properties": {
+            "div_precision_increment": {
```

Comment:
we will discuss this, usually we don't use underscore in property names. But since we are mapping MySQL configurations here, probably it's ok.